Pages/Cart-page/cart.css

```diff
@@ -0,0 +1,305 @@
+    margin:0 3rem 2rem 0;
+    cursor: pointer;
+}
+
+.badge-portion i{
+ 
+    color: var(--color-primary);
+}
+
+.badge-portion:hover{
+    background-color: var(--color-primary-opacity); 
+}
+
+.inp-btn{
+    width:40rem;
+    height:4rem;
+    background-color: var(--color-primary-opacity);
+    margin-right: 3rem;
+    margin-top:1rem;
+   
+}
+
+.cart-heading
+{
+    text-align: center;
+    margin-top:20rem;
+    color:var(--color-primary-darker);
+}
+
+.main-container{
+    margin-top:5rem;
+    margin-left: 5rem;
+    display: flex;
+    flex-direction: row;
+}
+
+.purchased-card{
+    display: flex;
+    flex-direction: column;
+    row-gap:5rem;
+}
+
+
+
+
+/* CARD */
+
+
+.entire-card{
+    display: grid;
+    grid-template-columns: 22rem auto;
+    border-radius:1rem;
+    background-color: var(--color-white);
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+}
+
+
+.entire-img{
+    max-width:20rem;
+    max-height:20rem;
+}
+
+.entire-img img{
+    width:100%;
+    height: 100%;
+}
+.column-two{
+    padding:1rem 1rem 0 0;
+}
+
+.card-text h4{
+    color:var(--light-grey);
+}
+
+.card-text p{
+    margin-top:1rem;
+    color:var(--color-primary-darker);
+    font-weight: 900;
+}
+
+.card-text p span{
+    color:var(--light-grey);
+    text-decoration: line-through; 
+    font-size: 12px;
+    margin-right:5px;
+}
+
+.quantity-btns{
+    margin-top:1rem;
+    cursor: pointer;
+}
+
+.quantity-btns span{
+ 
+    border: 1px solid var(--light-grey);
+    padding:5px;
+    font-size:12px;
+    margin:0 1rem 0 1rem;
+
+}
+
+.quantity-btns button{
+    background-color: var(--color-primary);
+    color:var(--color-beige);
+    border: none;
+    font-size: 14px;
+    font-weight: 900;
+    cursor: pointer;
+}
+
+
+.foot-btns{
+    margin-top:5rem;
+
+}
+
+/* CHECK CARD */
+
+.check-container{
+    border-radius:1rem;
+    background-color: var(--color-white);
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
+    width:60rem;
+    max-height: 25rem;
+    padding:2rem 2rem 0 2rem;
+    margin:10rem 0 0 20rem;
+}
+
+
+.check-container h2{
+    border-bottom: 1px solid black;
+}
+
+
+.check-container .price{
+    display: flex;
+    justify-content: space-between;
+    margin:1rem 0 1rem 0;
+  
+}
+
+.check-container .discount{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 1rem;
+}
+
+
+.check-container .del-charges{
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 1rem;
+}
+
+
+.check-container .total{
+    display: flex;
+    justify-content: space-between;
+    border-top: 1px solid black;
+    border-bottom: 1px solid black;
+    padding:1rem 0 1rem 1rem;
+    margin-bottom: 1rem;
+}
```

Comment:
make a variable for black color as it used multiple times

Reply:
Okay. will make the required changes.Thanks @rdthetha